doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -72,15 +72,15 @@ In order for the Kubernetes components to be conformant with the Reference Archi
+|`ra2.k8s.001`|Kubernetes Conformance|The Kubernetes distribution, product, or installer used in the implementation **must** be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) and marked (X) as conformant for Kubernetes version 1.21.|[req.gen.cnt.03](./chapter02.md#23-kubernetes-architecture-requirements)||
 |`ra2.k8s.002`|Highly available etcd|An implementation must consist of either three, five or seven nodes running the etcd service (can be colocated on the master nodes, or can run on separate nodes, but not on worker nodes).|[req.gen.rsl.02 req.gen.avl.01](./chapter02.md#23-kubernetes-architecture-requirements)|["For the high availability requirement"](../../../ref_impl/cntt-ri2/chapters/chapter04.md#431-installation-on-bare-metal-infratructure)|
 |`ra2.k8s.003`|Highly available control plane|An implementation must consist of at least one master node per availability zone or fault domain to ensure the high availability and resilience of the Kubernetes control plane services. A master node must run at least the following Kubernetes control plane services: `kube-apiserver`, `kube-scheduler` and `kube-controller-manager`|[req.gen.rsl.02](./chapter02.md#23-kubernetes-architecture-requirements)<br>[req.gen.avl.01](./chapter02.md#23-kubernetes-architecture-requirements)|
 |`ra2.k8s.012`|Highly available control plane|A master node must run at least the following Kubernetes control plane services: `kube-apiserver`, `kube-scheduler` and `kube-controller-manager`|[req.gen.rsl.02](./chapter02.md#23-kubernetes-architecture-requirements)<br>[req.gen.avl.01](./chapter02.md#23-kubernetes-architecture-requirements)|
 |`ra2.k8s.004`|Highly available worker nodes|An implementation must consist of at least one worker node per availability zone or fault domain to ensure the high availability and resilience of workloads managed by Kubernetes|[req.gen.rsl.01](./chapter02.md#23-kubernetes-architecture-requirements)<br>[req.gen.avl.01](./chapter02.md#23-kubernetes-architecture-requirements)<br>[req.kcm.gen.02](./chapter02.md#23-kubernetes-architecture-requirements)<br>[req.inf.com.01](./chapter02.md#23-kubernetes-architecture-requirements)|
-|`ra2.k8s.005`|Kubernetes API Version|In alignment with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), an implementation must use one of three latest minor versions (`n-2`). e.g. if the latest version is 1.17 then the RI must use either 1.17, 1.16 or 1.15.|TBC|
+|`ra2.k8s.005`|Kubernetes API Version|In alignment with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), an implementation **must** use Kubernetes version 1.21.|TBC|
```

Comment:
instead of specifying 1.21 in multiple places can we refer to a previously specified Kubernetes version.

Reply:
- We define the version "only" in 3 places and I think it is more important to state clear requirements than not to repeat information
 - As we discussed in the RA2 meeting the support window of Kubernetes is longer than the hoped release cadence of Anuket, so it is enough to specify one Kubernetes release per Anuket release.
 - Other components in the architecture should just be compatible with the Kubernetes release if otherwise not specified

Reply:
@CsatariGergely The same can be accomplished if the three places refer to the single source of truth.

Reply:
I've fixed this for Ch4 in 7cc06c0

Reply:
```suggestion
|`ra2.k8s.005`|Kubernetes API Version|In alignment with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), an implementation **must** use Kubernetes version as per the subcomponent versions table in [./README.md]|TBC|
```
propose to actually refer to the main table you've created

Reply:
Fixed in d7be782